app/javascript/components/miq-structured-list/index.jsx

```diff
@@ -81,7 +81,7 @@ const MiqStructuredList = ({
 
   /** Function to render the items of cell */
   const renderContent = (row) => (
-    <div className={classNames('content', row.bold ? 'label_header' : '')}>
+    <div className={classNames('content', row.bold ? 'label_header' : '')} style={row.style}>
```

Comment:
Is it possible to create different styles with different ids and set the div to a variable id name instead of using the in line style? Just trying to see if we can avoid in line styling.

Reply:
yes, here it's possible to assign a class name and use CSS to avoid inline style.